### ISSUE TITLE
Easily Broken switch

### ISSUE THREAD

**Issue by kicker22004** [jfm92/microByte_PCB#4]
My unit arrived yesterday and was damaged from heat exposure, I've left you a comment on crowdsource with this information, upon reprinting the cases and getting the unit back together the Right toggle feel off the switch taking the inner plunger with it so that switch is going to need to be replaced now, I'm going to redesign the button so that it fits inside the case with two lips instead of one so that the button doesn't need glued to the switch head. I'm hoping that my other one doesn't fail also. And on further inspection my power switch is bent down I'm not sure if it's from heat or the pressure from the case but i'll be replacing it and making changes to the case if needed.

**Comment by redfox67** [comment]
I had a similar issue, the right switch was not working on arrival. Overconfident in my making skills I tried to fix it and broke the switch. That is a great idea to do a redesigned button, if you do get that done could I use that file? Also do you know the switches name/number?

**Comment by kicker22004** [comment]
> I had a similar issue, the right switch was not working on arrival. Overconfident in my making skills I tried to fix it and broke the switch. That is a great idea to do a redesigned button, if you do get that done could I use that file? Also do you know the switches name/number?

https://lcsc.com/mobile/product-detail/Tactile-Switches_ALPS-Electric-SKSCLCE010_C139799.html
That is the link to the switch, also it seems there was already a button redesign 3 months ago. I printed the new design yesterday but can't test until my switches get here. I know the other switch is going to break trying to remove the printed button so I ordered 5 switches just in case.

**Comment by redfox67** [comment]
Thank you, how hard do you think it would be to desolder the switch as opposed to cutting it off?

**Comment by kicker22004** [comment]
For me it is very simple but I have a bit of experience working with SMD.

**Comment by redfox67** [comment]
Yeah, I ended up crushing the switch then removed it, however when I tried soldering a replacement button I had on hand the continuity check showed that there was a bridge between the control wire and ground. After desoldering and trying to fix it , it still is linked to ground. either I tell the firmware that one of the menu buttons is the right switch or I route a different pin on the u6 to the button which is an even more difficult soldering job. What would you recommend I do?

**Comment by kicker22004** [comment]
I would remove your switch and then test the pads, those shouldn't be touching, after that test the switch off the board as well.

**Comment by redfox67** [comment]
I tried testing the pads, what I believe to be the problem is that the pad that is connected to the u6 is soldered to the sheet of copper in the pcb that is grounded.

**Comment by kicker22004** [comment]
Oh that's unfortunate, I'd reach out to the creator but if you damaged it then it's on you.

**Comment by jfm92** [comment]
Hi!

Sorry for taking so long to follow up on this issue. 
The U6 is solder the cooper plain to dissipate heat, to unsolder it I recommend using a heat gun instead of an iron solder, it can be very challenging :) 
Anyways, I don't fully understand the relation with switch issue and the U6? Are you trying to use the PCB buttons to switch the power?
If it so, I don't recommend you to do so, because the switch just connect the battery circuit with the rest of the rest of PCB so it's just like join to wires.
![image](https://user-images.githubusercontent.com/13625726/127030856-6b752586-6c55-4647-92d5-bdd140fd6c00.png)
![image](https://user-images.githubusercontent.com/13625726/127031027-5a8ebaf9-7e10-4075-bcb5-6d98db5e4292.png)

**Comment by redfox67** [comment]
Thank you for responding, the issue is the right bumper/trigger thing, What happened is in the schematic the switch is between ground and pin 6 of U6, or the controls IC and if I run a continuity check of those two solder pads they are connected without there even being a switch, which means that in any game that uses that button it will be permanently triggered.

**Comment by redfox67** [comment]
![microByte_PCBTOP](https://user-images.githubusercontent.com/53539117/127037692-6bcd9dd3-5d4a-4640-84dc-dc2880e4f1d6.png)
